driver/collector/collector_factory.py

```diff
@@ -253,9 +253,16 @@ def get_collector(
             collector = MysqlCollector(conn, version)
         elif driver_conf["db_type"] in ["postgres", "aurora_postgresql"]:
             pg_conf = create_db_config_postgres(driver_conf)
-            conn = connect_postgres(pg_conf)
-            version = get_postgres_version(conn)
-            collector = PostgresCollector(conn, version)
+            conns: Dict[str, Any] = {}
+            for logical_database in driver_conf["postgres_db_list"]:
+                pg_conf_logical = pg_conf.copy()
```

Comment:
I guess "pg_conf_logical = pg_conf.copy()" could be move out of for loop?

Reply:
Correct

Reply:
Done